R/correlation.R

```diff
@@ -472,6 +491,18 @@ Correlation <- function(jaspResults, dataset, options){
 
       result$vsmpr <- jaspBase:::VovkSellkeMPR(result$p.value)
       result$vsmpr <- ifelse(result$vsmpr == "∞", Inf, result$vsmpr)
+
+      #effect size (fisher's z) and SE are computed following the recommendations of
+      #Fieller, E. C., Hartley, H. O., & Pearson, E. S. (1957). Tests for rank correlation coefficients: I. Biometrika, 44, 470–481.
+      result$effect.size <- atanh(result$estimate)
+      if(method == "pearson")
+        result$se.effect.size <- sqrt(1/(sample.size-3))
+      if(method == "spearman")
+        result$se.effect.size <- sqrt(1.06/(sample.size-3))
+      if(method == "kendall")
+        result$se.effect.size <- sqrt(0.437/(sample.size-4))
```

Comment:
does this work like that for partial correlations as well?

I think at least for the pearson coefficient, it should be something like `sqrt(1/(sample.size - 3 - number.of.controlling.variables))`. Not sure how it works for the spearman and kendall coefficient. In the worst case you can just show a message that SE for this or that case is not available.